src/Simulation/EntryPointDriver.Tests/Tests.qs

```diff
@@ -5,308 +5,317 @@
 // No Options
 //
 
-namespace Microsoft.Quantum.EntryPointDriver.Tests {
+// --- Returns Unit
```

Comment:
It would be useful to add a comment at the start of this file describing that `// ---` is used as a separator for test cases and that the string that follows the separator is interpreted as the name of the test case.